include/ntop_defines.h

```diff
@@ -790,6 +790,11 @@
 #define DEFAULT_HTTP_INDEX_PAGE \
   ""
 
+#define CONST_PREFS_CH_EXPORT_PATH \
+  NTOPNG_PREFS_PREFIX ".export_flows_path"
```

Comment:
The key is a bit too generic, I suggest a more descriptive one, e.g. clickhouse_flows_archive_path